gui/wxpython/gui_core/wrap.py

```diff
@@ -238,7 +238,10 @@ def __init__(self, *args, **kwargs):
 
     def SetToolTip(self, tip):
         if wxPythonPhoenix:
-            wx.StaticText.SetToolTip(self, tipString=tip)
+            if isinstance(tip, wx.ToolTip):
+                wx.StaticText.SetToolTip(self, tip=tip)
+            else:
+                wx.StaticText.SetToolTip(self, tipString=tip)
```

Comment:
Looking more at this, shouldn't it be enough to call:
```
wx.StaticText.SetToolTip(self, tip)
```
the overloading in wx should take care of the types.

Reply:
> the overloading in wx should take care of the types.

Yes, you are right, overload will solve it.